test/ConfigDistributor.spec.js

```diff
@@ -197,4 +155,126 @@ describe('A DistributeConfig class', () => {
 
   });
 
+  context('when actually interacting with the file system', () => {
+
+    let data;
+    let invalidName;
+    let validName;
+
+    beforeEach(() => {
+      data = 'some data';
+      invalidName = (function () {
```

Comment:
does `invalidName` need to be so complicated? If it's a directory, shouldn't `'./directory-that-does-not-exist'` carry out the same job?

Reply:
checked the implementation, and this attempts to create an error by using a too long file name; whereas a 255-byte limit is quite common, this may be filesystem dependent

Reply:
In this case we can't use the absence of a directory as a failure case, because `recursive: true` at https://github.com/libero/libero-pattern-library-config-manager/blob/master/bin/ConfigDistributor.js#L110 forces creation of a directory if it doesn't exist (which is desired).